pydm/widgets/channel.py

```diff
@@ -127,8 +123,6 @@ def disconnect(self, destroying=False):
         """
         Disconnect a PyDMChannel
         """
-        if is_qt_designer() and not config.DESIGNER_ONLINE:
```

Comment:
I removed this from here because now we could have connections regardless if `DESIGNER_ONLINE` is True, but I think I should be adding a check here to see if the `designer_online_by_default = True` somehow?

@hhslepicka what do you think?

Reply:
There is no need to add the check here since it is now delegated to the Data Plugin class. The channel won't be added to the list so the remove connection won't be a problem.